free-3.md

```diff
@@ -1,49 +1,79 @@
 # Free project 3
 
+
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
-
 ### What's the need?
-_What need is met by your idea? Who are you helping? What is that person's
-experience like now? What would their experience be like if you could help 
-them?_
+_What need is met by your idea?_
+
+I want to meet the need of having more convenient keyboard shortcuts for arranging music in MuseScore.
 
+_Who are you helping? What is that person's experience like now?_
 
+Many people who love music and want to compose music use a software like MuseScore, which can notate, on a musical staff, the notes that someone plays on the piano keyboard. However, not everyone owns a piano keyboard! 
+
+_What would their experience be like if you could help them?_
+
+Right now, users have to either purchase a special keyboard, or suffer through inputting very few notes at a time onto MuseScore. My DSL could greatly improve the efficiency of writing music.
+ 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL would be appropriate for my users because what they lack is the translation from more convenient laptop keyboard inputs into MuseScore. The domain specificity of this language is important; this language does not have to perform the function of a general programming language, but what it should do is provide just enough functionality for users to translate their computer keyboard strokes into MuseScore dication. If the language was not domain specific, the learning curve might be unnecessarily steep for the function/goal of the language, and might involve a lot of unnecessary elements that are unrelated to the user’s goals. 
 
 ### Why you?
-_What excites you about this idea? How did you come up with it?_
 
+_What excites you about this idea?_
+
+I really enjoy arranging music, and would really enjoy a tool that makes the process more efficient! 
+
+_How did you come up with it?_
+
+I came up with it one day when I was really frustrated with the painstaking process of translating musical thoughts into sheet music via MuseScore, and I did an angry keyboard smash. Then, I realized how cool it would be if I could tell Musecore the tempo of my song, and then press the computer keyboard keys that correspond to the notes I wanted to dictate. Ideally, the composition software would also compare how long I hold the key down (lets say 700 milliseconds) to the BPM (tempo) of the tempo I determined, to notate not only the notes themselves but the duration of the notes.
 
```

Comment:
This is a neat idea, that I would characterize as surprising, yet obvious.  After all, our laptop has a "keyboard," so why can't it double as a piano keyboard?